.pipelines/pypl-publishing.yml

```diff
@@ -37,38 +37,13 @@ parameters:
 - name: enable_linux_rocm
   displayName: 'Whether Linux ROCm package is built.'
   type: boolean
-  default: true
+  default: false
 
 - name: enable_macos_cpu
   displayName: 'Whether MacOS CPU package is built.'
   type: boolean
   default: true
 
-- name: ort_version
```

Comment:
These versions are no long used after @baijumeswani 's previous work. The ORT downloads are handled in cmake.